app/models/manageiq/providers/network_manager.rb

```diff
@@ -1,10 +1,12 @@
 module ManageIQ::Providers
   class NetworkManager < BaseManager
+    include SupportsFeatureMixin
     class << model_name
       define_method(:route_key) { "ems_networks" }
       define_method(:singular_route_key) { "ems_network" }
     end
 
+    supports_not :create
```

Comment:
ah, it can't be create, call it ems_network_new

it has to be an existing feature from https://github.com/manageiq/manageiq/blob/master/db/fixtures/miq_product_features.yml#L3843

otherwise it won't pass CI, change all occurences